.github/workflows/ci.yml

```
@@ -39,14 +82,379 @@ jobs:
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-shiny-${{ hashFiles('**/pyproject.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install Shiny and dev deps
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e '.[dev]'
+          pip install shiny
+      - name: Download Playwright browsers artifact
+        uses: actions/download-artifact@v3
```

actionlint (1.7.9)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)